<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QLabel" name="_shortcutLabel">
     <property name="text">
      <string>Shortcut</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="captureWindowCmdTextBox"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="_captureWindowShortcutLabel">
     <property name="text">
      <string>Shortcut</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="testHostButton">
     <property name="text">
      <string>Test Connection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_screenshotCmdLabel">
     <property name="text">
      <string>Capture Area Command</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="secretKeyTextBox"/>
   </item>
   <item row="9" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="_secretKeyLabel">
     <property name="text">
      <string>Secret Key</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="accessKeyTextBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_imgRepoLabel">
     <property name="text">
      <string>Evidence Repository</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="_hostPathLabel">
     <property name="text">
      <string>Host Path</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="screenshotCmdTextBox"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="eviRepoBrowseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_accessKeyLabel">
     <property name="text">
      <string>Access Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="hostPathTextBox"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="eviRepoTextBox"/>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="_captureWindowLabel">
     <property name="text">
      <string>Capture Window Command</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLineEdit" name="screenshotShortcutTextBox"/>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLineEdit" name="captureWindowShortCutTextBox"/>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QLabel" name="statusIconLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eviRepoTextBox</tabstop>
  <tabstop>eviRepoBrowseButton</tabstop>
  <tabstop>accessKeyTextBox</tabstop>
  <tabstop>secretKeyTextBox</tabstop>
  <tabstop>hostPathTextBox</tabstop>
  <tabstop>screenshotCmdTextBox</tabstop>
  <tabstop>screenshotShortcutTextBox</tabstop>
  <tabstop>captureWindowCmdTextBox</tabstop>
  <tabstop>captureWindowShortCutTextBox</tabstop>
  <tabstop>testHostButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
